doc/user/pmd.rst

```diff
@@ -0,0 +1,205 @@
+keyword.
+
+.. _pmd-commands:
+
+PMd Commands
+============
+
+.. index:: pm
+.. clicmd:: pm
+
+   Opens the PM daemon configuration node.
+
+.. index:: session <A.B.C.D|X:X::X:X> [{interface IFNAME|local-address <A.B.C.D|X:X::X:X>|vrf NAME}]
+.. clicmd:: session <A.B.C.D|X:X::X:X> [{interface IFNAME|local-address <A.B.C.D|X:X::X:X>|vrf NAME}]
+
+   Creates and configures a new PM session to send packets to and receive packets from.
+
+   `local-address` provides a local address that we should bind our
+   peer listener to and the address we should use to send the packets.
+
+   `interface` selects which interface we should use.
+
+   `vrf` selects which domain we want to use.
+
+.. index:: no session <A.B.C.D|X:X::X:X>$session [{local-address <A.B.C.D|X:X::X:X>$local|interface IFNAME$ifname|vrf NAME$vrfname}]
+.. clicmd:: no session <A.B.C.D|X:X::X:X>$session [{local-address <A.B.C.D|X:X::X:X>$local|interface IFNAME$ifname|vrf NAME$vrfname}]
+
+    Stops and removes the selected session.
+
+.. index:: show pm [vrf NAME] sessions [json]
+.. clicmd:: show pm [vrf NAME] sessions [json]
+
+    Show all configured PM sessions information and current status.
+
+.. index:: show pm [vrf NAME$vrfname] session <A.B.C.D|X:X::X:X>$peer [json]
+.. clicmd:: show pm [vrf NAME$vrfname] session <A.B.C.D|X:X::X:X>$peer [json]
+
+    Show status for a specific PM session.
+
+
+.. _pm-session-config:
+
+Session Configurations
+----------------------
+
+.. index:: [no] packet-size (1-65535)
+.. clicmd:: [no] packet-size (1-65535)
+
+   Configures the packet size of packets to send within sessions.
+   Default value is 80 bytes.
+
+.. index:: [no] packet-tos (1-255)
+.. clicmd:: [no] packet-tos (1-255)
+
+   The type of service value to be written in the appropriate
+   DSCP field of IP packet. Default value is 0xc0 which stands
+   for Inter Network Control value.
+
+.. index:: [no] interval (1-65535)
+.. clicmd:: [no] interval (1-65535)
+
+   Interval between each emission. This value is the interval
+   in milliseconds between each emission of packet. Default
+   interval value is set to 5000 ms.
+
+
+.. index:: [no] timeout (1-65535)
+.. clicmd:: [no] timeout (1-65535)
+
+   Timeout value expressed in milliseconds. This is the time
+   after emission, where one considers that the packet is not
+   received in time. That value can not be greater than
+   interval value. Default value is set to 5000 ms.
+
+
+.. index:: [no] retries down-count (1-255) up-count (1-255)
+.. clicmd:: [no] retries down-count (1-255) up-count (1-255)
+
+   When packets are not received, or not received in time,
+   the pm session is considered as lost. Reversely, the
+   reception of one packet in time makes the pm session go up.
+   However, it is possible to make more flexible the flip-
+   flap algorithm by increasing the retry timers. Those timers
+   permit to consider the session to go up or down, only after
+   a defined amount of received or non received retries.
```

Comment:
default value for this? default value * default interval should be our default timeout.